### PR TITLE
Github PR Template

### PR DESCRIPTION
**Pull request by rsdmike**
Adding simplistic PR template to assist reviewers with context when reviewing a Pull Request

Signed-off-by: Mike Johanson <michael.johanson@intel.com>

### CODE REVIEW

**Review by cloudxxx8 — DISMISSED** [review]
LGTM
please rebase this PR and I will merge it
Thanks

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+## PR Checklist
+Please check if your PR fulfills the following requirements:
```
Comment (lenny-goodell) [inline .github/PULL_REQUEST_TEMPLATE.md:3]: This is the short list for edgex-docs. Please update with full checklist.

**Review by lenny-goodell — CHANGES_REQUESTED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (cloudxxx8) [reply .github/PULL_REQUEST_TEMPLATE.md:3]: @rsdmike could you please update this PR according to Lenny' request?  Thanks.

Comment (cloudxxx8) [reply .github/PULL_REQUEST_TEMPLATE.md:3]: @rsdmike could you please help resolve this PR?

**Review by lenny-goodell — APPROVED** [review]
LGTM

**Pull request merged**